beacon-chain/rpc/prysm/v1alpha1/validator/proposer_capella.go

```diff
@@ -0,0 +1,25 @@
+package validator
+
+import (
+	"github.com/prysmaticlabs/prysm/v3/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/v3/config/params"
+	"github.com/prysmaticlabs/prysm/v3/consensus-types/interfaces"
+	"github.com/prysmaticlabs/prysm/v3/time/slots"
+)
+
+// Sets the bls to exec data for a block.
+func (vs *Server) setBlsToExecData(blk interfaces.BeaconBlock, headState state.BeaconState) error {
```

Comment:
I think it's better to deal with the error case withing this function. That is this function does not return an error, and if the call to `SetBLSToExecutionChanges` errors out, we set the empty slice here.

Reply:
Agree